app/model/editions/EditionsFront.scala

```diff
@@ -1,14 +1,22 @@
 package model.editions
 
+import org.postgresql.util.PGobject
 import play.api.libs.json.Json
 import scalikejdbc.WrappedResultSet
 
-case class EditionsFrontMetadata(nameOverride: Option[String])
-
 object EditionsFrontMetadata {
   implicit val format = Json.format[EditionsFrontMetadata]
 }
 
+case class EditionsFrontMetadata(nameOverride: Option[String], swatch: Option[Swatch]) {
+  def toPGobject: PGobject = {
+    val pgo = new PGobject()
+    pgo.setType("json")
+    pgo.setValue(Json.toJson(this).toString())
+    pgo
+  }
+}
+
```

Comment:
This is nice - we could make a generic along the lines of `def toPGobject[T](o: T)(implicit writes: Writes[T])` so we avoid duplicating this boilerplate everywhere.